Move 1 2

MacejkoTi opened the pull request.

Commit:
Flag field C3 and D2

We can be sure that these fields (C3 and D2) are not clean.

Commit:
Clear H9 field

mentor-tara commented:
Looks like something is wrong :astonished:	Please see the CI checks below for more information. Click the "Details" link for more information as shown in this screenshot.

![Screenshot of CI checks](https://media.graphcms.com/qz1AmyOsQWCWNh8kcuFw)[](error)

Commit:
Fix JSON structure

There was broken JSON structure in command.json field.

Commit:
Fix command 4

This command said `Clear H9` and board is only 8x8. For ceratin,
this command enters field out of range.

mentor-tara reviewed (approved):
### You're good. No mines hit. Merge this PR to continue the game.

This is the current status:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** |   |   |   |   |   |   |   |   |
| **2** |   |   |   | :pirate_flag: | 2 | 2 | 3 | 2 |
| **3** |   |   | :pirate_flag: | 2 | 1 | 0 | 0 | 0 |
| **4** | 1 | 2 | 1 | 1 | 0 | 0 | 0 | 0 |
| **5** | 0 | 0 | 0 | 0 | 0 | 0 | 1 | 1 |
| **6** | 0 | 0 | 0 | 1 | 1 | 1 | 2 |   |
| **7** | 0 | 1 | 1 | 2 |   |   |   |   |
| **8** | 0 | 1 |   |   |   |   |   | 1 |

Mines remaining: 8
Moves made: 4[](clear-H8)
